raiden/blockchain/events.py

```diff
@@ -30,7 +31,13 @@
 )
 from raiden_contracts.contract_manager import ContractManager
 
-EventListener = namedtuple("EventListener", ("event_name", "filter", "abi"))
+
+@dataclass(frozen=True)
+class EventListener:
+    event_name: Optional[str]
```

Comment:
Why is this optional?

Reply:
Somehow I thought it could be set to `None`, but I was wrong. Changed it to `str`.